### PR TITLE
12.0 mig mail_tracking_mailgun

### PR DESCRIPTION
**Pull request by ernestotejeda**
Depends on #318 
Cc @Tecnativa

### CODE REVIEW

**Comment by hbrunn** [comment]
something with your [dependencies](https://travis-ci.org/OCA/social/jobs/451147983#L925) seems wrong

**Comment by hbrunn** [comment]
sorry, didn't read the first message...

**Comment by pedrobaeza** [comment]
Please rebase

**Comment by ernestotejeda** [comment]
rebased

File: mail_tracking_mailgun/README.rst [hunk 1]
```diff
@@ -1,115 +1 @@
...
-   :target: https://odoo-community.org
-
-This module is maintained by the OCA.
-
-OCA, or the Odoo Community Association, is a nonprofit organization whose
-mission is to support the collaborative development of Odoo features and
-promote its widespread use.
-
-To contribute to this module, please visit https://odoo-community.org.
+**This file is going to be generated by oca-gen-addon-readme.**
```
Comment (pedrobaeza) [inline mail_tracking_mailgun/README.rst]: Please generate this file

**Review by pedrobaeza — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

**Pull request merged**